internal/stage_common.go

```diff
@@ -36,7 +36,7 @@ func getFirstLine(s string) string {
 
 func logFriendlyHTTPMessage(logger *logger.Logger, msg string, logPrefix string) {
 	for _, line := range strings.Split(msg, "\r\n") {
-		logger.Debugf("%s %s", logPrefix, line)
+		logger.Debugf("%q %q", logPrefix, line)
```

Comment:
@libmartinito hard to tell if this is appropriate without seeing an example log

Reply:
Hmmm...this one definitely should not use `%q`